environments/exercise2.r

```diff
@@ -12,3 +12,17 @@ myGet <- function(x) {
   NULL
 }
 
+recursiveGet <- function(name, env = parent.frame()) {
+  stopifnot(is.character(name), length(name) == 1)
+  if (identical(env, emptyenv())) {
+    # base case
+    warning(paste(name," not found"))
+    NULL
+  } else if (is.element(name,ls(env))) {
+    # success case
+    env[[`name`]]
```

Comment:
@rylandely had backticks in his ```myGet```. I just followed his style here.
I guess I'll remove them in both functions